src/DynamoCoreWpf/ViewModels/PackageManager/Docs/PublishPackageDocumentation.html

```diff
@@ -0,0 +1,226 @@
+        color: #ce9178;
+    }
+
+    code .v {
+        color: #b5cea8;
+    }
+
+    code .o {
+        color: #c586c0;
+        font-weight: bold;
+    }
+
+    code .i {
+        color: #569cd6;
+        font-weight: bold;
+    }
+
+    code .b {
+        color: #9cdcfe;
+        font-weight: bold;
+    }
+
+    code .e {
+        color: #43c9b0;
+    }
+
+    code > .nl .nl {
+        padding-left: 10px;
+    }
+
+    code .nl {
+        display: block;
+    }
+
+    thead {
+        text-align: center;
+    }
+
+    .hidden-lg {
+        display: none;
+    }
+
+    .button {
+        background-color: transparent;
+        border-color: #3c3c3c;
+        color: #bbbbbb;
+    }
+
+    .button:hover {
+        border-color: #656565;
+        background-color: #373737;
+        opacity: 1.0;
+    }
+
+    @media screen and (max-width: 600px) {
+        table {
+            width: 100%;
+        }
+
+        table,
+        th,
+        td {
+            border: none;
+        }
+
+        tr {
+            border: 1px solid white;
+            display: block;
+        }
+
+        td,
+        th {
+            display: block;
+        }
+
+        .hidden-lg {
+            display: block;
+            font-weight: bold;
+            text-align: center;
+            border-bottom: 1px solid white;
+        }
+
+        .hidden-sm {
+            display: none;
+        }
+
+        .spaced-sm {
+            margin-top: 20px;
+        }
+
+        tbody tr:first-of-type {
+            border-bottom: none;
+        }
+    }
+</style>
+
+<!-- Use inline styles to keep Dynamo Dictionary styling in the dictionary -->
+<!-- and custom styling in Dynamo sidepanel. -->
+
+<body style="background-color: #353535; font-family: 'Artifakt Element', 'Open Sans';">
+    <h3>Publishing Dynamo Packages</h3>
+    <p>
+        Dynamo offers a vast no. of features out of the box, along with those we also maintain an extensive package library,
+        contributed by the Dynamo community, these can significantly extend Dynamo's capability. This documentation describes how a user can publish a package from Dynamo.
+    </p>
+    <p>
+        The menu option to publish a package can be found at: "Packages > Publish New Package...",
+        the dialog box that opens up after it will ask for all the information related to the package the user wants to publish.
+    </p>
+    <h4>Package Name:</h4>
+    <p>
+        Name of the package you want to publish. This will appear in the library and on the package manager website.
+    </p>
+    <p>
+        <b>Description:</b><br />
+        This is a great place to add more details about your package, you can list any exceptions and pre-requisites.
+        This can be used to educate users on what the package does and how it can be utilised into their workspaces.
+    </p>
+    <p>
+        <b>Version:</b><br />
+        The version field is bifurcated into major/minor/build sections as per the versioning semantics.
```

Comment:
This is quite technical! I would suggest we simplify a little, as below:

The version field is split into three sections, major, minor and build, which allows you as a package author to upload new versions of your package that follow versioning semantics - Users may want to publish more versions for their packages with new and improved features, polish some corners which were left out in the first version, or fix a bug.